threshsign/include/threshsign/eddsa/EdDSAMultisigVerifier.h

```diff
@@ -17,6 +17,8 @@
 
 class EdDSAMultisigVerifier;
 
+using concord::crypto::openssl::eddsa::EdDSAVerifier;
```

Comment:
no using statements in headers

Reply:
Removed in https://github.com/arc-vmware/concord-bft/pull/6/commits/87f4edd0f4f5d72b422cef50f4a8eeeee8b28bc2